ci-operator/step-registry/cucushift/installer/check/capability/cucushift-installer-check-capability-commands.sh

```diff
@@ -84,6 +84,7 @@ caps_operator[Storage]="storage"
 caps_operator[NodeTuning]="node-tuning"
 caps_operator[MachineAPI]="machine-api control-plane-machine-set cluster-autoscaler"
 caps_operator[ImageRegistry]="image-registry"
+caps_operator[OperatorLifecycleManager]="operator-lifecycle-manager"
```

Comment:
this capability includes several operators:
```
caps_operator[OperatorLifecycleManager]="operator-lifecycle-manager operator-lifecycle-manager-catalog operator-lifecycle-manager-packageserver"
```

Reply:
Updated.